docs/4-enable-audio.md

```diff
@@ -0,0 +1,144 @@
+# Getting Started with the Nexmo Conversation JS SDK
+
+In this getting started guide we'll cover adding audio events to the Conversation we created in the [simple conversation with events](3-inviting-members.md) getting started guide. We'll deal with media events, the ones that come via the conversation, and the ones we send to the conversation.
+
+## Concepts
+
+This guide will introduce you to the following concepts.
+
+- **Audio** - enabling and disabling Audio streams in a Conversation
+- **Media Events** - `member:media` events that fire on a Conversation when media state changes for a member
+
+## Before you begin
+
+- Ensure you have run through the [previous guide](3-utilizing-events.md)
+
+## 1 - Update the JavaScript App
+
+We will use the application we already created for [the third getting started guide](3-utilizing-events.md). All the basic setup has been done in the previous guides and should be in place. We can now focus on updating the client-side application.
+
+### 1.1 - Add audio UI
+
+First, we'll add the UI for user to enable and disable audio, as well as an `<audio>` element that we'll use to play the Audio stream from the conversation. Let's add the UI at the top of the messages area.
+
+```html
+<section id="messages">
+  <div>
+    <audio id="audio">
+      <source>
+    </audio>
+    <button id="enable">Enable Audio</button>
+    <button id="disable">Disable Audio</button>
```

Comment:
Just an idea: A radio toggle instead of buttons?

Reply:
that's not native in HTML sadly, it would involve quite [some more HTML and CSS](https://www.w3schools.com/howto/howto_css_switch.asp) and I think that would steal focus from the audio part of the QS